File: android/Xtrade/app/src/main/java/nz/co/fendustries/xtrade/adapters/viewHolders/ConvertedRateViewHolder.java
package nz.co.fendustries.xtrade.adapters.viewHolders;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import nz.co.fendustries.xtrade.R;

/**
 * Created by joshuafenemore on 21/10/16.
 */
public class ConvertedRateViewHolder extends RecyclerView.ViewHolder
{
    private ImageView forexFlagImageView;
    private TextView forexCodeTextView;
    private TextView sellRateTextView;

    public ConvertedRateViewHolder(View itemView)
    {
        super(itemView);
        this.forexFlagImageView = (ImageView) itemView.findViewById(R.id.forexFlagImageView);
        this.forexCodeTextView = (TextView) itemView.findViewById(R.id.forexCodeTextView);
        this.sellRateTextView = (TextView) itemView.findViewById(R.id.sellRateTextView);
    }

    public ImageView getForexFlagImageView()
    {
        return forexFlagImageView;
    }

    public TextView getForexCodeTextView()
    {
        return forexCodeTextView;
    }

    public TextView getSellRateTextView()
    {
        return sellRateTextView;
    }
}

File: xamarin/Xtrade/Xtrade.AndroidApp/Adapters/ViewHolders/ConvertedRateViewHolder.cs
namespace Xtrade.AndroidApp.Adapters.ViewHolders
{
    using Android.Support.V7.Widget;
    using Android.Views;
    using Android.Widget;

    public class ConvertedRateViewHolder : RecyclerView.ViewHolder
    {
        public ImageView ForexFlagImageView { get; set; }

        public TextView ForexCodeTextView { get; set; }

        public TextView SellRateTextView { get; set; }

        public ConvertedRateViewHolder(View itemView) : base(itemView)
        {
            this.ForexFlagImageView = itemView.FindViewById<ImageView>(Resource.Id.forexFlagImageView);
            this.ForexCodeTextView = itemView.FindViewById<TextView>(Resource.Id.forexCodeTextView);
            this.SellRateTextView = itemView.FindViewById<TextView>(Resource.Id.sellRateTextView);
        }
    }
}